tests/e2e/env/setup.sh

```diff
@@ -274,5 +312,25 @@ mkdir -p $WCP_ROOT/screenshots
 echo "Disabling rate limiter for card declined in E2E tests"
 cli wp option add wcpay_session_rate_limiter_disabled_wcpay_card_declined_registry yes
 
+# Log test configuration for visibility
+echo
+echo "*******************************************************"
+echo "Current test configuration"
+echo "*******************************************************"
+
 echo
+echo "WordPress version:"
+cli_debug wp core version
+
+echo
+echo "WooCommerce version:"
+cli_debug wp plugin get woocommerce --field=version
+
+echo
+echo "Docker environment:"
+cli_debug wp cli info
```

Comment:
Do we need this to be logged as well? The other two make sense but not sure if this adds value.